docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -112,7 +110,52 @@ To send a command from the client and to process it on the server side:
     }
     ```
 
-<a name="servermetadata"></a>
+## Updating and Retreiving Playlist
+
+To create a playlist from the server side:
+
+1. Create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+    ```
+    var playlist = new MediaControlPlaylist("MyFavorite");
+    ```
+
+You can create playlist with Metadata also like below code.
+
+    ```
```

Comment:
Indentation for this code block is not accurate. Kindly fix the same.

Reply:
Done